projects/tabs/src/modules/tabs/tab-button.component.ts

```diff
@@ -60,8 +60,23 @@ export class SkyTabButtonComponent {
     }
   }
 
-  public onButtonEnterKeyDown(event: any): void {
-    this.onButtonClick(event);
+  public onTabButtonKeyDown(event: KeyboardEvent): void {
+    /*istanbul ignore else */
+    if (event.key) {
+      switch (event.key.toLowerCase()) {
```

Comment:
I believe that @Blackbaud-PaulCrowder pointed out recently that we should always use `toUpperCase` instead of `toLowerCase` when doing string checks. In this case I don't think it would matter since we aren't dealing with international text - but probably worth just fixing for consistency.